src/flat-button.jsx

```diff
@@ -265,6 +265,7 @@ const FlatButton = React.createClass({
       userSelect: 'none',
       position: 'relative',
       overflow: 'hidden',
+      verticalAlign: 'middle',
```

Comment:
What about adding this property at this line https://github.com/callemall/material-ui/blob/master/src/enhanced-button.jsx#L287.
We could do something like:
```js
verticalAlign: other.hasOwnProperty('href') ? 'middle' : null,
```